pkg/app/piped/cloudprovider/lambda/client.go

```diff
@@ -87,9 +117,184 @@ func (c *client) Apply(ctx context.Context, fm FunctionManifest) error {
 				return fmt.Errorf("total code size per account exceeded: %w", err)
 			case lambda.ErrCodeResourceNotFoundException, lambda.ErrCodeResourceNotReadyException:
 				return fmt.Errorf("resource error occurred: %w", err)
+			case lambda.ErrCodeTooManyRequestsException:
+				return fmt.Errorf("request throughput limit was exceeded: %w", err)
 			}
 		}
 		return fmt.Errorf("unknown error given: %w", err)
 	}
 	return nil
 }
+
+func (c *client) UpdateFunction(ctx context.Context, fm FunctionManifest) error {
+	codeInput := &lambda.UpdateFunctionCodeInput{
+		FunctionName: aws.String(fm.Spec.Name),
+		ImageUri:     aws.String(fm.Spec.ImageURI),
+	}
+	_, err := c.client.UpdateFunctionCodeWithContext(ctx, codeInput)
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			case lambda.ErrCodeInvalidParameterValueException:
+				return fmt.Errorf("invalid parameter given: %w", err)
+			case lambda.ErrCodeServiceException:
+				return fmt.Errorf("aws lambda service encountered an internal error: %w", err)
+			case lambda.ErrCodeCodeStorageExceededException:
+				return fmt.Errorf("total code size per account exceeded: %w", err)
+			case lambda.ErrCodeTooManyRequestsException:
+				return fmt.Errorf("request throughput limit was exceeded: %w", err)
+			case lambda.ErrCodeResourceConflictException:
+				return fmt.Errorf("resource already existed or in progress: %w", err)
+			}
+		}
+		return fmt.Errorf("unknown error given: %w", err)
+	}
+
+	// TODO more configurable fields from here:
```

Comment:
you know what, actually Kapetanios's TODO plugin uses only `TODO:` or `FIXME:` as a keyword by default. So you'd better use `TODO:` if you need to open a new issue.
https://kapetanios.dev/docs/plugins/todo/